packages/react-router/modules/matchPath.js

```diff
@@ -35,7 +35,8 @@ function matchPath(pathname, options = {}) {
   const paths = [].concat(path);
 
   return paths.reduce((matched, path) => {
-    if (!path) return null;
+    // https://github.com/ReactTraining/react-router/issues/6941
```

Comment:
```suggestion
```
The link to this will be in their history, so this comment isn't needed.

Reply:
removed